test-resources/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "one-login-test-resources",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "deploy-test-resources-to-dev": "cd ../helper-scripts && ./deploy_test_resources.sh",
+    "format": "./node_modules/.bin/prettier --write '**/*.ts'",
+    "format:check": "./node_modules/.bin/prettier --check '**/*.ts'",
+    "lint": "./node_modules/.bin/eslint ./testDequeue",
+    "test:dequeue": "./node_modules/.bin/jest --testPathPattern=testDequeue/ --silent"
```

Comment:
take it or leave it: keep this as test:unit for now, until we have more assets in this directory?

Reply:
Updated